### PR TITLE
added multiclass-mcc

### PR DESCRIPTION
**Pull request by behrica**

### CODE REVIEW

File: src/confuse/multi_class_metrics.clj [hunk 1]
```diff
@@ -67,3 +67,40 @@
   (let [tpisum (apply + (mapv (partial b/true-positives actual predicted) classes))
         fnisum (apply + (mapv (partial b/false-negatives actual predicted) classes))]
     (double (/ tpisum (+ tpisum fnisum))))))
+
+
+
+(defn- t [cm k]
```
Comment (shark8me) [inline src/confuse/multi_class_metrics.clj]: a more descriptive name would be nice, perhaps `row-sum` ? Similarly `column-sum` for line 76?

**Comment by shark8me** [comment]
Thanks for the PR!
A descriptive fn name would be nice to have, other than that, looks good to me

File: src/confuse/multi_class_metrics.clj (hunk 1 above)
Comment (behrica) [reply src/confuse/multi_class_metrics.clj]: they come from the MCC formula in the wikipedia, maybe they should become local functions.
It is the first time I converted some maths to Clojure, so I  am open to best practices...

**Comment by behrica** [comment]
I did not change the names, but made the functions local.
They make indeed only sense in the context of MCC and by looking at the formula in wikipedia.

Not sure, if I should still change the names.

**Comment by behrica** [comment]
I did an other commit, replacing the names by something "longer".
Not sure, if better, as it makes the "formula" harder to read.

Maybe decide yourself, which you prefer.

**Comment by behrica** [comment]
There was an edge case where we get a division by 0.
In this case teh MCC is consiered 0.

sklearn does the same.

**Comment by shark8me** [comment]
Looks good to me, thanks for the PR!

**Pull request merged**